sdk/authorization/arm-authorization-profile-2019-03-01-hybrid/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@azure/arm-authorization-profile-2019-03-01-hybrid",
+  "author": "Microsoft Corporation",
+  "description": "AuthorizationManagementClient Library with typescript type definitions for node.js and browser.",
```

Comment:
should we mention in the end `for 2019-03-01-hybrid profile`?

Reply:
I think the name is clear enough.

Reply:
Have added the profile as one of the keywords manually for now and also updated the homepage to reflect the right folder.